draft-ietf-oauth-status-list.md

```diff
@@ -117,7 +118,7 @@ Status List Token:
 : A token in JWT or CWT representation that contains a cryptographically secured Status List.
 
 Referenced Token:
-: A token in JWT or CWT representation which contains a reference to a Status List or Status List Token. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
+: A cryptographically secured container which contains a reference to a Status List or Status List Token. It is RECOMMENDED to use secured containers based on JSON Object Signing as defined in {{RFC7515}} and {{RFC7519}} or CBOR Object Signing and Encryption as defined in {{RFC8152}}. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
```

Comment:
I'm unsure about the word container, maybe token is better

Reply:
I agree I think token is better.

Reply:
Maybe "A cryptographically secured data structure"

Reply:
```suggestion
: A cryptographically secured data structure which contains a reference to a Status List or Status List Token. It is RECOMMENDED to use JSON or CBOR for representation of the token and secure it using JSON Object Signing as defined in {{RFC7515}} or CBOR Object Signing and Encryption as defined in {{RFC8152}}. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
```